### PR TITLE
added challenge: Lending Challenge on Aave

### PR DESCRIPTION
**Pull request by Turupawn**

### CODE REVIEW

File: src/challenges/defi-challenges/simple-lending.mdx [hunk 1]
```diff
@@ -0,0 +1,221 @@
+---
+name: Simple Lending
+index: 3
+level: 3
+website: "https://github.com/LevelUpWeb3/SimpleTokenSwap-Challenge"
+summary: Introduction to implementing a simple token swap contract by understanding Uniswap swap function.
```
Comment (emilyJLin95) [inline src/challenges/defi-challenges/simple-lending.mdx:6]: I think we need to change this metadata and GH repo?

**Comment by Turupawn** [comment]
@emilyJLin95 @PrintRH I updated the challenge by making some corrections and adding an explanation on how to run unit the test. I feel like this is ready to be merged, please help me reviewing :pray:

**Review by emilyJLin95 — COMMENTED** [review]
For the most part LGTM - do you have permissions to create the challenge repo under the levelupweb3 org?

**Comment by PrintRH** [comment]
@emilyJLin95 - I am not sure, but I don't think so. To keep things moving quicker, I have created the challenge repo here: https://github.com/LevelUpWeb3/SimpleLending-Challenge

**Comment by Turupawn** [comment]
@PrintRH Thanks for creating the repo, I added the code there and updated the challenge. Also run the `processContentMarkdown.js` and manually fixed the expected conflicts

File: src/challenges/defi-challenges/simple-lending.mdx [hunk 2]
```diff
@@ -0,0 +1,244 @@
+---
+name: Simple Lending
+index: 3
```
Comment (PrintRH) [inline src/challenges/defi-challenges/simple-lending.mdx:3]: Can we update this to index "4"? So, this follows after recent challenge from 0x.

File: src/challenges/defi-challenges/simple-lending.mdx [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+level: 3
+website: "https://github.com/Turupawn/LendingChallengeLevelUp"
+summary: Understand the inner workings of Aave by completing the Lending challenge.
+heroImage: "/images/level-up-preview.png"
+labels: ["DeFi"]
+---
+
+import ContentCopy from "src/components/ContentCopy";
```
Comment (PrintRH) [inline src/challenges/defi-challenges/simple-lending.mdx:11]: In this new update, ContentCopy integration is no longer required. You can directly write code in traditional markdown format.

File: src/challenges/defi-challenges/simple-lending.mdx [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+Your task is to:
+1. Develop your own contract with `deposit` and `withdraw` functionalities based on the provided base contract.
+2. Deploy it to Scroll Sepolia testnet.
+3. And finally verify it.
+ 
+If you need help with using a smart contract framework for completing this challenge, the [Level Up: Build with Foundry](https://www.levelup.xyz/content/level-up-foundry) guide might be a helpful start!
+
+If you get stuck, feel free to ask for help in [Level Up Telegram group](https://t.me/+PdNbk5milo1mMTAy).
+
+<ContentCopy code={`// SPDX-License-Identifier: MIT
```
Comment (PrintRH) [inline src/challenges/defi-challenges/simple-lending.mdx:32]: Remove ContentCopy component and can use the standard format writing code in markdown.

File: src/challenges/defi-challenges/simple-lending.mdx [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+If you haven't completed the Token Swap challenge yet, check it out [here](https://www.levelup.xyz/challenges/simple-token-swap)!
+
+Let's take a step further and dive into another core DeFi primitive, lending. Aave is one of the top protocols, securing billions in assets. Let's learn how to earn yield by lending through smart contracts.
+
+In this challenge, you'll implement a contract that deposits into Aave pools on your behalf. You'll also be able to withdraw your supplied tokens along with the earned rewards.
+
+## Objective
+
+Your task is to:
+1. Develop your own contract with `deposit` and `withdraw` functionalities based on the provided base contract.
```
Comment (PrintRH) [inline src/challenges/defi-challenges/simple-lending.mdx:24]: Challenge below seems to define `stake` and `withdraw` function. Should we update `deposit` to `stake`?

Also, I notice that the majority of the challenge calls out `supply` function. Does it make more sense to just use `supply` for rest of the challenge?

**Review by PrintRH — COMMENTED** [review]
@Turupawn - Challenge is coming along great! Left a few comments for fixes then I think we should be ready to ship this!

File: src/challenges/defi-challenges/simple-lending.mdx (hunk 5 above)
Comment (Turupawn) [reply src/challenges/defi-challenges/simple-lending.mdx:24]: The thing is the Aave functions are called `deposit` and `withdraw`, and I named the challenge functions `stake` and `withdraw` so they are different. But yeah that sounds confusing.

So I think we should rename the challenges functions to `proxyDeposit` and `proxyWithdraw` to make a clear distinction. Other option could be `stake` and `unstake`. What do you think?

Comment (PrintRH) [reply src/challenges/defi-challenges/simple-lending.mdx:24]: That makes sense! Personally, I think `stake` and `unstake` provides more clarity. I think then just updating the `deposit` and `withdraw` pair for the rest of this challenge will be sufficient, and we can ship it!

**Comment by emilyJLin95** [comment]
Oh @Turupawn 

Realized, for removing the existing swaps challenge, could we just add it as part of this PR since this is getting merged and reviewed now?

**Comment by Turupawn** [comment]
@PrintRH @emilyJLin95 
I addressed all comments + temporarily removed the lending challenge

**Comment by PrintRH** [comment]
@Turupawn Challenges shaping up nicely!

One last thing - can you also run the `build` command as a last push? I am noticing some dev fragments which might break some links/contents in prod.

**Comment by Turupawn** [comment]
@PrintRH I just run it, did it work?

**Review by PrintRH — APPROVED** [review]
It worked!

Changes LGTM! Let's ship this. Thank you for your hard work here @Turupawn !

**Comment by PrintRH** [comment]
Oh wait one last thing @Turupawn - merging is still blocked, can you pull and merge the latest changes from `main` to this branch?

That should resolve the merge error.

**Comment by Turupawn** [comment]
@PrintRH done1

**Pull request merged**